plugins/test/unit/plugins/importers/yum/test_sync.py

```diff
@@ -34,7 +34,7 @@
 
 class BaseSyncTest(unittest.TestCase):
     def setUp(self):
-        self.url = 'http://pulpproject.org/'
+        self.url = 'https://repos.fedorapeople.org/repos/pulp/pulp/demo_repos/zoo/'
```

Comment:
Do these tests live download from this URL? If so, we should not do that as I believe the tests currently pass without Internet access (I'm not 100% sure of that).